openslides/users/static/js/users/users.js

```diff
@@ -30,6 +36,14 @@ angular.module('OpenSlidesApp.users', [])
             views: {
                 '@users.user': {}
             }
+        })
+        .state('users.user.csv-import', {})
```

Comment:
You have to change this code to:
```javascript
        .state('users.user.csv-import', {
            url: '/csv-import',
            controller: 'UserCSVImportCtrl',
        })
```
And put it before the user detail state.

Reason:
*I added some code to set the default values for url and controller for the detail-, list-, create- and update-views. But I can not guess the values for other views.
* the detail view accepts any view that starts with /users/user/*, in other words it accepts ```/users/user/csv-import``` for that reason any state which url starts with /users/user/ has to be defined before the detail view. Alternatively you could read the documentation for ui-router and find a way, that the detail-view only accepts urls '/users/user/INTEGER